mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -154,6 +154,20 @@ func ResourceSqlDatabaseInstance() *schema.Resource {
 							Required:    true,
 							Description: `The machine type to use. See tiers for more details and supported versions. Postgres supports only shared-core machine types, and custom machine types such as db-custom-2-13312. See the Custom Machine Type Documentation to learn about specifying custom machine types.`,
 						},
+						"advanced_machine_features": {
```

Comment:
The API will always return nothing if the user has not explicitly sent a value, right?

Reply:
Yes, thats right!